docs/form-question-types.rst

```diff
@@ -138,6 +138,37 @@ The external app widget is displayed when the ``appearance`` attribute begins wi
   text,ex_string_widget,Ex string widget,ex:change.uw.android.BREATHCOUNT,text type with ex:change.uw.android.BREATHCOUNT appearance (can use other external apps)
 
 
+.. _masked-text-widget:
+
+Masking the entered value
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+type
+  ``text``
+appearance
+  ``masked``
+
+If ``masked`` is added to ``appearance``, the answers will be obscured with dots similar to password-protected fields.
```

Comment:
I think we should say something brief about what it's intended for. Maybe "This can be used to hide input as it is being entered but it does not provide any security measures beyond that."